getusers/getusers.py

```diff
@@ -512,19 +518,17 @@ def show_all_users(ARGS):
 
 
 def main():
-    '''Main script execution'''
-    show_header()
-    init_variables()
-
     PARSER = options()
     ARGS = PARSER.parse_args()
 
-    if ARGS.show_version:

```

Comment:
This introduces an upstream bug where the Version info command no longer works:

Before:
```
[root@rhel6 ~]# curl -s https://raw.githubusercontent.com/rackerlabs/useful-scripts/8324e1ab0c6f11d53bb05792da27cbf30e6a10ce/getusers/getusers.py | python - -v

   ______     __     __  __
  / ____/__  / /_   / / / /_______  __________
 / / __/ _ \/ __/  / / / / ___/ _ \/ ___/ ___/
/ /_/ /  __/ /_   / /_/ (__  )  __/ /  (__  )
\____/\___/\__/   \____/____/\___/_/  /____/

Showing version

Version: 1.0.0
```

Now:
```
[root@rhel6 ~]# curl -s https://raw.githubusercontent.com/rackerlabs/useful-scripts/dd2d6f39e51e8f71b997d2616441f838c5087812/getusers/getusers.py | python - -v

   ______     __     __  __
  / ____/__  / /_   / / / /_______  __________
 / / __/ _ \/ __/  / / / / ___/ _ \/ ___/ ___/
/ /_/ /  __/ /_   / /_/ (__  )  __/ /  (__  )
\____/\___/\__/   \____/____/\___/_/  /____/

Default: Showing standard users

ID     User    Home          Shell       Sudo  Last Login
1000   user1    /home/user1    /bin/bash   yes   Mon Mar 16 08:51:08 2020
1001    user2   /home/user2    /bin/bash   yes   None found
```